### ISSUE TITLE
[web]: webdev serve error protobuf-0.13.11

### ISSUE THREAD

**Issue by mslimani** [flutter/flutter#32313]
Hi, 

I have an error 'protobuf pubspec.yaml not found' when run : webdev serve.

Error : 
webdev could not run for this project.
Could not find a file named "pubspec.yaml" in "/Users/mehdisli/.pub-cache/hosted/pub.dartlang.org/protobuf-0.13.11".

**Comment by sunnyseeds** [comment]
Me three.

**Comment by TheJulianJES** [comment]
``flutter packages pub global run webdev build`` should work as a workaround.

**Comment by sunnyseeds** [comment]
> `flutter packages pub global run webdev build` should work as a workaround.

This worked. Thanks a lot!

**Comment by BrunoEleodoro** [comment]
> `flutter packages pub global run webdev build` should work as a workaround.

Many thanks bro.

**Comment by mslimani** [comment]
This works for me : replace `webdev serve` by `flutter packages pub global run webdev serve`

**Comment by kevmoo** [comment]
Duplicate of https://github.com/flutter/flutter/issues/32276

We need to improve the docs here!

**Comment by juanjoserodrigolazaro** [comment]
i add protobuf: ^0.13.11 to dependencies then in idle terminal run "pub get" then close and open your idle.

**Comment by dbacinski** [comment]
@kevmoo exporting PATH in a linked do not fix this issue. Only https://github.com/flutter/flutter/issues/32313#issuecomment-490523545 or https://github.com/flutter/flutter/issues/32313#issuecomment-490653337 works for me.

It seems that `webdev` is looking for protobuf: ^0.13.11 in cache for some reason, but it does not exist.

**Comment by richasay** [comment]
I'm trying `flutter packages pub global run webdev build` but I get an message that "Waiting for another flutter command to release the startup lock..." any ideas how to find the flutter process that is locking the webdev server from restarting?

**Comment by richasay** [comment]
> @kevmoo exporting PATH in a linked #32276 do not fix this issue. Only [#32313 (comment)](https://github.com/flutter/flutter/issues/32313#issuecomment-490523545) or [#32313 (comment)](https://github.com/flutter/flutter/issues/32313#issuecomment-490653337) works for me.
> 
> It seems that `webdev` is looking for protobuf: ^0.13.11 in cache for some reason, but it does not exist.

I manually added the protobuf dir and added a pubspec.yaml file within, but I have missing pieces of code within, but it got passed the error of missing pubspec file itself...

**Comment by SebastianBordi** [comment]
> This works for me : replace `webdev serve` by `flutter packages pub global run webdev serve`

thank you so much. thats work for me 
i was in you city 3 years ago and i fall in love

**Comment by yasinarik** [comment]
When I run 

`flutter packages pub global run webdev serve` or
`flutter packages pub global run webdev build` commands it gives this error:

```
No active package webdev.
pub finished with exit code 65
```

So I ran this command: `pub global activate webdev` and as it says it successfully installs and activates: 

```
Precompiled webdev:webdev.
Installed executable webdev.
Activated webdev 2.6.2.
```

Then I run the first commands again but it doesn't work. If I try to use VSCode `Start Debugging` or `Start Without Debugging` it complains like this:

```
Flutter web projects may fail to launch if they reference the Flutter SDK in pubspec.yaml
```

So I used `flutter upgrade` and even the very time consuming `flutter pub cache repair` commands. Nothing changed. I tried the `protobuf: ^0.13.11` . It didn't help too. 

I am very frustrated at the moment :(
